developer-extension/README.md

```diff
@@ -40,3 +40,58 @@ Then, in Google Chrome:
 ## Browser compatibility
 
 For now, only Google Chrome is supported.
+
+## Usage Tips
+
+### Event Tab
+
+The Event Tab contains a list of events sent by the SDK and a menu of event types for quick filtering.
+
+#### Search Syntax
+
+We support a basic `key:value` search syntax which means you can search within the limits of [RUM event structures](https://docs.datadoghq.com/real_user_monitoring/explorer/search/), such as `action.target.name:action_name`. While all the fields can be added as columns in the event list, it does not mean the existing ones are valid fields: the `Description` summary is not part of RUM event structure. You can click on any `Description` cell for details of the event to look for correct key to search with.
```

Comment:
**💬 suggestion:** ‏don't talk about the Description column in the "Search Syntax" section, as "columns" are different than "search", and it's not quite related to the syntax. I'm not sure if we should mention it at all.